projects/ion/src/lib/card/card.component.html

```diff
@@ -8,6 +8,28 @@
         data-testid="icon-title"
       ></ion-icon>
       <h4 data-testid="cardHeader">{{ configuration.header.title }}</h4>
+      <ion-icon
+        *ngIf="configuration.header.infoTooltip"
+        type="information"
+        class="icon-info"
+        data-testid="icon-info"
+        [size]="18"
+        ionTooltip
+        [ionTooltipTitle]="configuration.header.infoTooltip.ionTooltipTitle"
+        [ionTooltipPosition]="
+          configuration.header.infoTooltip.ionTooltipPosition
+        "
```

Comment:
These properties should have a default value, like:
```suggestion
        [ionTooltipPosition]="
          configuration.header.infoTooltip.ionTooltipPosition || 'bottomLeft'
        "
```
If you don't pass these properties on infoTooltip object, then the tooltip won't be able to render properly.

Reply:
thanks for the review🥰